packages/cmf/src/assert.js

```diff
@@ -1,16 +1,19 @@
-// eslint-disable-next-line import/prefer-default-export
-export function assertTypeOf(options, attr, type, toThrow = true) {
-	let isTypeCorrect;
-	if (options[attr] === undefined) {
+export function assertValueTypeOf(value, type) {
```

Comment:
This function does not throw. So any call of it is useless.
Please move the error here, instead of `assertTypeOf`